shared/js/abp-preprocessed.es6.js

```diff
@@ -116,11 +120,11 @@ function updateLists () {
 }
 
 // Make sure the list updater runs on start up
-settings.ready().then(() => updateLists())
+settings.ready().then(() => abpLists.updateLists())
 
 chrome.alarms.onAlarm.addListener(alarm => {
     if (alarm.name === 'updateLists') {
-        settings.ready().then(() => updateLists())
+        settings.ready().then(() => abpLists.updateLists())
```

Comment:
This change is in preparation for moving all the chrome event handlers to a separate file.